template.rb

```diff
@@ -80,7 +80,7 @@
 
 # Install development and test gems
 gem_group :development, :test do
-  gem 'wildland_dev_tools', '~>0.7.0', git: 'git+ssh://git@github.com/wildland/wildland_dev_tools.git'
+  gem 'wildland_dev_tools', git: 'git+ssh://git@github.com/wildland/wildland_dev_tools.git', :tag => 'v0.8.0'
```

Comment:
Well, don't use the old Ruby 1.8 style hash stuff with symbols. And you can just link it directly to github: `gem 'wildland_dev_tools', github: 'wildland/wildland_dev_tools' tag: 'v0.8.0'`